content/doc/book/pipeline/multibranch.adoc

```diff
@@ -0,0 +1,107 @@
+---
+layout: section
+---
+:notitle:
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:imagesdir: /doc/book/resources/pipeline
+:hide-uri-scheme:
+:toc:
+
+= Multibranch Pipelines
+
+In the <<jenkinsfile#, previous section>> a `Jenkinsfile` which could be
+checked into source control was implemented. This section covers the concept of
+*Multibranch* Pipelines which build on the `Jenkinsfile` foundation to provide
+more dynamic and automatic functionality in Jenkins.
+
+== Creating a Multibranch Pipeline
+
+The *Multibranch Pipeline* project type enables you to implement different
+Jenkinsfiles for different branches of the same project.
+In a Multibranch Pipeline project, Jenkins automatically discovers, manages and
+executes Pipelines for branches which contain a `Jenkinsfile` in source control
+
+This eliminates the need for manual Pipeline creation and management, as would
+be necessary when, for example, a developer adds a new feature branch.
+
+To create a Multibranch Pipeline:
+
+* Click *New Item* on Jenkins home page.
+
+image::new-item-selection.png["Click *New Item* on the Jenkins home page", role=center]
+
+* Enter a name for your Pipeline, select *Multibranch Pipeline* and click *OK*.
+
+[CAUTION]
+====
+Jenkins uses the name of the Pipeline to create directories on disk. Pipeline
+names which include spaces may uncover bugs in scripts which do not expect
+paths to contain spaces.
+====
+
+image::new-item-multibranch-creation.png["Enter a name, select *Multibranch Pipeline*, and click *OK*", role=center]
+
+* Add a *Branch Source* (for example, Git) and enter the location of the
+  repository.
+
+image::multibranch-branch-source.png["Add a Branch Source", role=center]
+image::multibranch-branch-source-configuration.png["Add the URL to the project repository", role=center]
+
+
+* *Save* the Multibranch Pipeline project.
+
+Upon *Save*, Jenkins automatically scans the designated repository and creates
+appropriate items for each branch in the repository which contains a
+`Jenkinsfile`.
+
+By default, Jenkins will not automatically reindex the repository for branch
```

Comment:
In a surprising deviation from my usual commentary, reindex should actually have an n-dash. :) This is optional, but recommended, since otherwise native English speakers will stumble over the two vowels side-by-side in reindex.

s/reindex/re-index